conans/client/conf/__init__.py

```diff
@@ -27,7 +27,7 @@
     # - For building cross compilation tools: Where the cross compiler will run.
     os:
         Windows:
-            subsystem: [None, cygwin, msys, msys2, wsl]
+            subsystem: [None, cygwin, msys, msys2, msys2_ucrt64, msys2_ming32, msys2_ming64, msys2_clang64, wsl]
```

Comment:
Maybe having both msys2 and msys could be a bit confusing? Because looks like the "base" subsystem for msys2 [is called msys](https://www.msys2.org/docs/environments/).

Reply:
I think `msys` is mostly dead, but we shouldn't get rid of the setting. Maybe in 2.0

Reply:
Yeah, tat ``msys`` intended to be to the previous msys2 version, which is mostly dead, but we probably shouldn't remove it, as we could break. Lets get rid of it in Conan 2.0